webpack/components/AnsibleHostDetail/components/AnsibleHostInventory/AnsibleHostInventory.js

```diff
@@ -0,0 +1,25 @@
+import React, { useMemo } from 'react';
+import Immutable from 'seamless-immutable';
+import './AnsibleHostInventory.scss';
+import ReportJsonViewer from "../../../ReportJsonViewer";
+import { foremanUrl } from 'foremanReact/common/helpers'
+import { useAPI } from 'foremanReact/common/hooks/API/APIHooks';
+
+const AnsibleHostInventory = ({ hostId }) => {
+    const params = useMemo(() => ({ params: { host_ids: [hostId] }}), [hostId]);
+
+    const url =  foremanUrl('/ansible/api/ansible_inventories/hosts');
+    const {
+        response: inventory,
+    } = useAPI('get', url, params);
```

Comment:
You could move the API call into the wrapper component and pass the inventory data as a prop. It will make the testing more simple as you will not need to mock the API call. More on testing below.